app/src/androidTest/java/com/github/se/polyfit/model/post/PostTest.kt

```diff
@@ -23,7 +23,7 @@ class PostTest {
   }
 
   @Test
-  fun `getCarbs returns correct nutrient when present`() {
+  fun getCarbsReturnsCorrectNutrientWhenPresent() {
```

Comment:
Maybe these changes could be done in a separate PR

Reply:
Due to the post class storing a `Uri` and the fact that this class does not exist in the unit tests, the test file had to be moved to the `androidTest` folder. In this folder, you cannot have function names with apostrophes or spaces, so all the function names in that file had to be renamed.